src/Backend/Core/Engine/DataGridFunctions.php

```diff
@@ -188,12 +188,19 @@ public static function getUser($id)
                 $html .= '     </a>' . "\n";
             }
             $html .= '  </div>';
-            $html .= '  <p><a href="' .
-                     BackendModel::createURLForAction(
-                         'edit',
-                         'users'
-                     ) . '&amp;id=' . $id . '">' . $nickname . '</a></p>' . "\n";
-            $html .= '</div>';
+            $html .= '  <p>';
+            if ($allowed) {
+                $html .= '<a href="' .
+                BackendModel::createURLForAction(
+                    'edit',
```

Comment:
Now you're busy changing this, I'd also change the 'edit' and 'users' to start with a capital letter :)